### PR TITLE
fixes #6750 - change constraint on apipie-rails gem so it can be updated to 0.2.x

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ gem "ancestry", "~> 2.0.0"
 gem 'scoped_search', '>= 2.7.0', '< 3.0.0'
 gem 'net-ldap'
 gem 'uuidtools'
-gem "apipie-rails", "~> 0.1.1"
+gem "apipie-rails", "~> 0.2"
```
Comment (isratrade) [inline Gemfile]: @domcleal, @inecas, can I merge this?

Comment (domcleal) [inline Gemfile]: Shouldn't it be `~> 0.2.0` to stay within 0.2.0-0.3.0?

Comment (isratrade) [inline Gemfile]: you're right, it's based on the last digit. commit updated.

**Comment by domcleal** [comment]
Thanks @isratrade, merged as 6592ce8 for Foreman 1.6.0.

**Pull request closed without merging**